spec/models/city_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe City, type: :model do
+  it 'can have many restaurants' do
+    restaurant_count = 2
+    # Note: same as
+    # city = FactoryGirl.create(:city, :with_restaurants, restaurant_count: restaurant_count)
+    #
+    # If you take a look at spec/support/factory_girl, that's where we get this
+    # nice syntatic sugar from.
+    city = create(:city, :with_restaurants, restaurant_count: restaurant_count)
+    expect(city.restaurants).not_to be_nil
```

Comment:
If you put a `debugger` or `binding.pry` here, you will see that what FactoryGirl is creating is a literally an instance of the city model. Not only that, it has populated the city's restaurants relationship with two restaurants, and those restaurants have names generated by the Faker gem.